packages/net/tcp_connection.pony

```diff
@@ -775,16 +766,13 @@ actor TCPConnection
     This occurs only with IOCP on Windows.
     """
     ifdef windows then
-      match len.usize()
-      | 0 =>
+      if len.usize() == 0 then
```

Comment:
seems like this usize() probably isn't needed but I wanted to do the minimal change.